eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -73,7 +73,7 @@ public function loadBinaryFile($binaryFileId)
     {
         $this->checkBinaryFileId($binaryFileId);
 
-        if ($binaryFileId[0] === '/') {
+        if ($binaryFileId[0] === '/' || (PHP_OS === 'WINNT' && $binaryFileId[1] === ':')) {
```

Comment:
@andrerom, @bdunogier: I investigated all the test cases related to `loadBinaryFile` method and it's true that it applies to http paths. However... there's no test data containing an absolute http path (starting with `/`). 

So I see there three possible solutions:

1. Decide that [this test case](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/SPI/Tests/FieldType/ImageIntegrationTest.php#L282-L314) is wrong and remove it.
2. Consider that this test case might be written explicitly to test if it fails for erroneous data (maybe someone in the past tried to put there an absolute filesystem path and this test case was created to check if we're protected against that?).
3. Fix the test case so that an absolute http path is tested, not the filesystem path.

After reading your comments and discussing it during daily I'm leaning towards option no. 3, but need confirmation @andrerom, @bdunogier :)